src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+
+const convertToObject = (stylesString) => {
+  const styleObject = {};
+
+  const declarations = stylesString.split(';');
+
+  declarations.forEach((declaration) => {
+    const trimmedDeclaration = declaration.trim();
+
+    if (trimmedDeclaration) {
+      const [property, value] = trimmedDeclaration.split(':');
+
+      if (property && value) {
+        styleObject[property.trim()] = value.trim();
+      }
+    }
+  });
```

Comment:
[CODE STYLE] - When chaining methods, they should have the same indentation to improve readability. Additionally, consider using 'map' to transform each declaration, and 'reduce' to accumulate the results into an object, instead of 'forEach' which does not return a new array.